packages/nlopt/nlopt.0.7.0/opam

```diff
@@ -24,6 +24,7 @@ depends: [
 conflicts: [
   "base-nnp"
   "ocaml-option-nppchecker"
```

Comment:
Typo, should it be `ocaml-option-nnpchecker`? Although if nlopt.0.7.0 is supposed to work on OCaml 5.0, then these 2 conflicts should be dropped from this version though.